WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -1831,6 +1831,7 @@ ZmMailMsgView.prototype._renderMessageBody1 = function(params, part) {
         // calendar part in ICS format
         else if (ct === ZmMimeTable.TEXT_CAL) {
             content = ZmMailMsg.getTextFromCalendarPart(bp);
+            content = AjxStringUtil.htmlEncode(content);
```

Comment:
Change in AjxStringUtil._getOriginalHtmlContent  is not required, here.
For XSS issues, we usually encode the corresponding malicious fields.
AjxStringUtil._getOriginalHtmlContent  is subsequently called in further function calls.